uncrustify.cfg

```diff
@@ -55,14 +59,19 @@ sp_pp_concat				= ignore
 sp_pp_stringify				= ignore
 sp_bool						= force
 sp_after_class_colon    	= force
-sp_before_class_colon   	= remove
+sp_before_class_colon   	= force
+sp_after_constr_colon		= force
 sp_cmt_cpp_start			= add
 sp_cmt_cpp_doxygen			= true
+sp_endif_cmt				= force
 sp_angle_shift				= remove
```

Comment:
@evsh:  Again, why is the spacing different for these four, compared to the four lines above them?

Reply:
Err.. Pardon me, I can't see the difference.

Reply:
@evsh:  It looks like lines 64-67 are spaced 1-tab to the right of (the spacing of) lines 60-63.